packages/fx-core/templates/plugins/resource/simpleauth/bicep/simpleAuth.template.bicep

```diff
@@ -0,0 +1,64 @@
+param sku string
+param simpleAuthServerFarmsName string
+param simpleAuthWebAppName string
+param tenantId string
+param AADClientId string
+@secure()
+param AADClientSecret string
+param applicationIdUri string
+{{#contains 'fx-resource-frontend-hosting' plugins}}
+
```

Comment:
remove the extra line break?

Reply:
This line is needed to separate input parameters from dependent plugins for readability.